modules/gltf/editor/editor_scene_importer_blend.cpp

```diff
@@ -134,7 +134,19 @@ Node *EditorSceneFormatImporterBlend::import_scene(const String &p_path, uint32_
 
 	// Get global paths for source and sink.
 	// Escape paths to be valid Python strings to embed in the script.
-	const String source_global = ProjectSettings::get_singleton()->globalize_path(p_path).c_escape();
+	String source_global = ProjectSettings::get_singleton()->globalize_path(p_path);
+
+#ifdef WINDOWS_ENABLED
+	// On Windows, when using a network share path, the above will return a path starting with "//"
+	// which once handed to Blender will be treated like a relative path. So we need to replace the
+	// first two characters with "\\" to make it absolute again
+	if (source_global.is_network_share_path()) {
+		source_global = source_global.replace(0, 2, "\\");
```

Comment:
There's no such overload for strings, you'd need to instead:
```suggestion
		source_global = "//" + source_global.right(-2);
```
To strip the first two characters

Reply:
Oh! This is amazing, thanks so much. My cpp experience is mostly embedded and I'm new here so I need the help.

Reply:
To see details see `core/string/ustring.h`